inspirehep/modules/records/jsonschemas/records/experiments.json

```diff
@@ -5,14 +5,27 @@
             "title": "Accelerator",
             "type": "string"
         },
-        "affiliation": {
-            "description": "Institution ICN",
-            "title": "Affiliation",
-            "type": "array"
-        },
-        "affiliation_record": {
-            "$ref": "elements/json_reference.json",
-            "title": "Institution record URI"
+        "affiliations": {
+            "items": {
+                "properties": {
+                    "curated_relation": {
+                        "title": "Is the relation curated?",
```

Comment:
I don't think this `title` adds anything to the property name. In fact, IMO, most `title`s in schemas should go away.